keras/layers/convolutional/conv1d.py

```diff
@@ -75,8 +75,8 @@ class Conv1D(Conv):
         any `dilation_rate` value != 1.
       padding: One of `"valid"`, `"same"` or `"causal"` (case-insensitive).
         `"valid"` means no padding. `"same"` results in padding with zeros
-        evenly to the left/right or up/down of the input such that output has
-        the same height/width dimension as the input.
+        evenly to the left/right or up/down of the input.When padding=
```

Comment:
move padding to the next line and make it like `padding="same"`. Format the line lengths for the rest of the lines accordingly.